melodic/distribution.yaml

```diff
@@ -8346,6 +8346,22 @@ repositories:
       url: https://github.com/ros-perception/slam_karto.git
       version: melodic-devel
     status: maintained
+  slam_toolbox:
+    doc:
+      type: git
+      url: https://github.com/SteveMacenski/slam_toolbox.git
+      version: melodic-devel
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/SteveMacenski/slam_toolbox-release.git
+      version: 0.4.0-1
```

Comment:
So this is released as version 0.4.0, but I see in the upstream repository that there are tags for much later numbers, i.e. there is a 1.0.0.  While that doesn't really matter for this to work on the buildfarm, it is strange to say the least.  Can I recommend that you remove/deprecate the 0.4.0 tag and use a number > 1.0.0 for the version?  At least that way it will be internally consistent.

Reply:
Yeah, so the 1.0.0 was just a release, but there's no bloom or package.xml version along side it. The 1.0.0 is there for another entity to use as a static version for some features they were looking for. 

I can redo this for a > 1.0.0 number if you like, nbd